src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -181,13 +183,17 @@ private void WriteDevRuntimeConfig(ProjectContext projectContext)
 
         private void AddAdditionalProbingPaths(RuntimeOptions runtimeOptions, ProjectContext projectContext)
         {
-            foreach (var packageFolder in projectContext.LockFile.PackageFolders)
+            if (runtimeOptions.AdditionalProbingPaths == null)
             {
-                if (runtimeOptions.AdditionalProbingPaths == null)
-                {
-                    runtimeOptions.AdditionalProbingPaths = new List<string>();
-                }
+                runtimeOptions.AdditionalProbingPaths = new List<string>();
+            }
+
+            //Add developer profile store location as the first path to probe
+            var developerProfileStore = DeveloperStoreLocation +  "/.dotnet/store/|arch|/|tfm|";
```

Comment:
This feels odd that we are passing part of the path in on a parameter named `DeveloperStoreLocation`, and then the C# code appends more folders to the path.

Instead, I would expect the whole path to be built in one location (the MSBuild targets), and then passed in.  That way if, for whatever reason, someone can modify the whole path, if necessary.

Also, maybe it would be more extensible to have the parameter be "AdditionalProbingPaths" and take a list of paths.  Then the MSBuild targets just passes in one, but we can add more later (or someone else can).  Thoughts?